e2e/cypress/integration/team/teams_spec.js

```diff
@@ -182,4 +182,50 @@ describe('Teams Suite', () => {
         // * Ensure user is logged out
         cy.url({timeout: TIMEOUTS.LARGE}).should('include', 'login');
     });
+
+    it('TS14632 Team Icon should be Removed', () => {
+        cy.apiLogin('sysadmin');
+        cy.apiCreateTeam('test-team', 'Test Team').then(() => {
+            cy.visit('/');
+        });
```

Comment:
Please move this into a separate spec file (probably name as `remove_team_icon_spec.js`) so we can leverage before hook and get team from response body.  Also, instead of using `sysadmin`, you may create a new user so that it doesn't need to clean up after this test.  Like,
```javascript
let team;

describe('Teams Suite', () => {
    before(() => {
        // # Login as new user
        cy.loginAsNewUser().then(() => {
            // # Create new team and visit its URL
            cy.apiCreateTeam('test-team', 'Test Team').then((response) => {
                team = response.body;
                cy.visit(`/${team.name}`);
            });
        });
    });

    it('TS14632 Remove Team Icon', () => {...});
});
```